packages/preview/school-document/0.1.0/typst.toml

```diff
@@ -0,0 +1,14 @@
+[package]
+name = "school-document"
```

Comment:
I feel like the name of the package is a bit too generic. Can you please change it to someone more distinctive?

Reply:
> I feel like the name of the package is a bit too generic. Can you please change it to someone more distinctive?

Thanks for your feedback, I'll do so asap